microsoft-edge/visual-studio/index.md

```diff
@@ -34,52 +34,49 @@ The simplified workflow allows you to debug JavaScript that run in Microsoft Edg
 
-:::image type="complex" source="./media/set-web-browser.png" alt-text="Choose the preview channel of Microsoft Edge that you want Visual Studio to launch" lightbox="./media/set-web-browser.png":::
-   Choose the preview channel of Microsoft Edge that you want Visual Studio to launch  
+:::image type="complex" source="./media/set-web-browser.png" alt-text="Select the preview channel of Microsoft Edge that you want Visual Studio to launch" lightbox="./media/set-web-browser.png":::
+   Select the preview channel of Microsoft Edge that you want Visual Studio to launch  
 :::image-end:::  
 
-> [!NOTE]
-> If you choose Microsoft Edge \(EdgeHTML\), Visual Studio launches it instead of Microsoft Edge \(Chromium\).  [Install the one of the preview channels of Microsoft Edge][MicrosoftedgeinsiderDownload] or ensure that the version of Microsoft Edge installed on your machine is Microsoft Edge \(Chromium\) and not Microsoft Edge \(EdgeHTML\).  
-
-Now that Visual Studio is correctly configured, choose the green **Play** button.  Visual Studio builds your app, start the web server, launch Microsoft Edge, and navigate to `https://localhost:44362/` or whatever port is specified in `launchSettings.json`.  
+Now that Visual Studio is correctly configured, select the green **Play** button.  Visual Studio builds your app, starts the web server, launches Microsoft Edge, and navigates to `https://localhost:44362/` or whatever port is specified in `launchSettings.json`.  
 
 :::image type="complex" source="./media/edge-launch.png" alt-text="Microsoft Edge launches from Visual Studio" lightbox="./media/edge-launch.png":::
    Microsoft Edge launches from Visual Studio  
 :::image-end:::  
 
 ### Debug JavaScript running in Microsoft Edge  
 
-Switch back to Visual Studio.  In `Counter.js`, to set a breakpoint on Line 13, choose the gutter next to the line.  
+Switch back to Visual Studio.  In `Counter.js`, to set a breakpoint on Line 13, select the gutter next to the line.  
 
-:::image type="complex" source="./media/set-breakpoint.png" alt-text="Choose the gutter next to Line 13 in Counter.js to set a breakpoint in Visual Studio" lightbox="./media/set-breakpoint.png":::
-   Choose the gutter next to Line 13 in `Counter.js` to set a breakpoint in Visual Studio  
+:::image type="complex" source="./media/set-breakpoint.png" alt-text="Select the gutter next to Line 13 in Counter.js to set a breakpoint in Visual Studio" lightbox="./media/set-breakpoint.png":::
+   Select the gutter next to Line 13 in `Counter.js` to set a breakpoint in Visual Studio  
 :::image-end:::  
 
-Now switch back to the instance of Microsoft Edge that Visual Studio launched.  Choose on **Counter** in the NavMenu on the left of the webpage.  Now choose **Increment**.  
+Now switch back to the instance of Microsoft Edge that Visual Studio launched.  Select **Counter** in the navigation menu on the left of the webpage.  Now select **Increment**.  
```

Comment:
```suggestion
1.  Switch back to the instance of Microsoft Edge that Visual Studio launched.

1.  Select **Counter** in the navigation menu at the top of the webpage, and then select the **Increment** button.
```
Check the location of "Counter" UI, in context.    Looks like it's in the upper right in the screenshot.
Style guide says "don't state the UI control type, unless it helps clarity".  My retort is, it _always_ helps clarity, esp given the Accessibility rule, "don't assume the screenshots are visible to the audience".  I omit "the ... button" only in the easy case of the OK button.